open_lm/main.py

```diff
@@ -699,7 +716,7 @@ def main(args):
         loss = CrossEntropyLossWithZLoss(args.z_loss_coefficient)
 
     if args.dataset_manifest:
-        log_num_checkpoints(total_steps, args)
+        log_num_checkpoints(total_steps, args, shard_shuffle_seed)
```

Comment:
here `shard_shuffle_seed` should be in args so probably no need to change the api?

Reply:
Same reasoning as above (but I can also just move shard_shuffle_seed inside args).

Reply:
Moved `shard_shuffle_seed` in args.